### DISCUSSION TITLE
Managing charge settings with vehicles with different battery types

### DISCUSSION THREAD

**Discussion by alexeiw123** [ngardiner/TWCManager#543] [General]
At the moment I've got a model 3 with an NMC battery, so TWC manager is set up to charge to 80% for the usual scheduled and track green energy charging.

We're considering getting a model Y with LFP, which are suited to 100% charging.

Is there a feature to achieve this per car that I've missed?

**Comment by MikeBishop** [comment]
There is currently no feature to automatically apply different limits to different cars.  You could have TWCManager *not* set the limit, in which case the limits set on the cars would govern and could very well be different.
